Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -492,7 +494,8 @@ private func createResolvedPackages(
         // Establish dependencies in each target.
         for targetBuilder in packageBuilder.targets {
             // Record if we see a duplicate target.
-            foundDuplicateTarget = foundDuplicateTarget || !allTargetNames.insert(targetBuilder.target.name).inserted
+            foundDuplicateTarget = foundDuplicateTarget || !allTargetNames.insert(targetBuilder.target.name.lowercased()).inserted
```

Comment:
I wonder if it warrants an extension adding a new `lowercasedName` or `normalizedName` property to `Target` or `ResolvedTarget`?